py/circuitpy_mpconfig.mk

```diff
@@ -141,6 +141,9 @@ CFLAGS += -DCIRCUITPY_AUDIOCORE_DEBUG=$(CIRCUITPY_AUDIOCORE_DEBUG)
 CIRCUITPY_AUDIOMP3 ?= $(call enable-if-all,$(CIRCUITPY_FULL_BUILD) $(CIRCUITPY_AUDIOCORE))
 CFLAGS += -DCIRCUITPY_AUDIOMP3=$(CIRCUITPY_AUDIOMP3)
 
+CIRCUITPY_AUDIODELAYS ?= 0
```

Comment:
```suggestion
CIRCUITPY_AUDIOEFFECTS ?= 0
CIRCUITPY_AUDIODELAYS ?= $(CIRCUITPY_AUDIOEFFECTS)
```

Reply:
Missed keeping `audioeffects` around the first time but pushed and updated correctly now.